src/main.tf

```diff
@@ -4,7 +4,7 @@ resource "google_compute_network" "main" {
   auto_create_subnetworks = false
 
   depends_on = [
-    module.apis
+    google_project_service.main
```

Comment:
These would need to wait on the `null_resource.waiter`. Some lore around this:

The `google_project_service` resource is supposed to wait until the API is actually enabled (I think checked via polling). However, we were seeing situations where it would complete, then the next resource would provision (which required the API) and we would get an API enablement error. Thus we wait an extra 30s to really make sure the API is enabled.

I would love to get rid of the null_resource if we can, but it would require us checking if the `google_project_service` is waiting properly. Otherwise, if we are going to unroll the module we need to comment as to why the `null_resource` is there, and make sure the depends_on are pointed at it.